src/pages/guides/getting-started/creating-your-first-plugin/index.md

```diff
@@ -0,0 +1,182 @@
+---
+title: Creating your first plugin
+description: Create your first own Photoshop plugin using UXP.
+contributors:
+  - https://github.com/pklaschka
+---
+
+# Creating your first UXP-based Photoshop plugin
+
+In this tutorial, you will create your first own Photoshop plugin using UXP.
+
+## Prerequisites
+
+To follow this tutorial, you need **Photoshop v23 or above** and the **UXP Developer Tool (UDT)**. You can
+install both applications with the Creative Cloud Desktop app. You also need **basic knowledge of HTML and JavaScript**.
+
+## Step 1: Create the plugin
+
+Open the UXP Developer Tool and click the _"Create plugin..."_ button. In the dialog that opens, enter the following
+details:
+
+| Field                    | Value           |
+| ------------------------ | --------------- |
+| Plugin Name              | My first plugin |
+| Plugin Id                | my.first.plugin |
+| Plugin Version           | 1.0.0           |
+| Host Application         | Adobe Photoshop |
+| Host Application Version | 23.0.0          |
+| Template                 | ps-starter      |
+
+Click _"Select Folder"_ and select a new, empty folder where you want to develop your plugin. The Developer Tool creates
+the necessary files in your selected folder and adds the plugin to the list of plugins.
+
+You should now be able to see the following files in your selected folder:
+
+- `icons`: a folder containing a set of icons for your plugin
+- `index.html`: your plugin's entrypoint. This gets loaded when the plugin loads
+- `manifest.json`: a file containing the metadata you specified in UDT as well as a pointer to the `index.html` as an
+  entrypoint.
+
+As well as some other "meta" files like `LICENSE`, `README.md`, etc.
+
+## Step 2: Run the plugin
+
+Open Adobe Photoshop and create some sample file you can use with your plugin.
+
+To run your plugin within Photoshop, click the button on the right of your plugin's row within the UDT and select _"
+Load"_. This loads your plugin in Photoshop and automatically opens the plugin's panel.
+
+The _ps-starter_ template initially contains a panel with a _"Populate Layers"_ button that allows you to populate a box
+within the panel with the layer names:
+
+![A Photoshop panel with a list of layer names and a button called "Populate Layers"](panel-base.png)
+
+Let's take a look at what's going on in the background when running this plugin.
+
+In the `manifest.json` file, the `"main"` field points to the `index.html` file:
+
+```json
+{
+  "id": "my.first.plugin",
+  "name": "My first plugin",
+  "version": "1.0.0",
+  "main": "index.html",
+  "host": [
```

Comment:
In our samples, let's not use `[]` for `host`, since that'll get rejected at marketplace submission time.

We should update the sample to remove this, actually.

Reply:
Waiting to resolve via UDT sample updates.